Assets/_BForBoss/_Weapons/Scripts/Bullets/BulletBehaviour.ApplyDamage.cs

```diff
@@ -10,6 +10,8 @@ protected void HitObject(Collider col, Vector3 hitPosition, Vector3 hitNormal)
             if(col.TryGetComponent(out LifeCycleBehaviour lifeCycle))
             {
                 lifeCycle.Damage(_properties.Damage);
+                var isDead = !lifeCycle.IsAlive;
```

Comment:
If it is for readability and all that's fair, but not sure if it's worth caching a local variable if it's only being used once?

Reply:
Yeah sure